src/common/addremovectrl.cpp

```diff
@@ -54,7 +54,7 @@ wxAddRemoveCtrl::Create(wxWindow* parent,
         return false;
 
     // We don't do anything here, the buttons are created when we're given the
-    // adaptor to use them with in SetAdaptor().
+    // adaptor to use them within SetAdaptor().
```

Comment:
This looks wrong too. The old for seems more logical: `buttons` [...] `we're given the adaptor to use them with` [where?] `in SetAdaptor()`. IOW "the buttons are created in SetAdaptor() when we're given the adaptor to use them with"

Reply:
Maybe the word "in" should be stricken?  One of the words should be stricken at least.